recipe/meta.yaml

```diff
@@ -37,12 +36,14 @@ requirements:
     - matplotlib >=2.0.0
     - networkx >=1.8
     - pillow >=4.3.0
-    - dask-core >=0.15
     - toolz >=0.7.4
     - pywavelets >=0.4.0
     - dask-core >=0.9.0
     - cloudpickle >=0.2.1
     - imageio >=2.1.0
+    # scikit-image depends on dask-array
+    # which requires numpy, dask-core and toolz (cytoolz for speed)
+    - cytoolz >=0.7.3
```

Comment:
I don't know whether it's a good idea include Cytoolz... Doesn't this go against your philosophy of making a minimal package?

Reply:
I'm mostly for "having choice to create a minimal package if I want to". I don't care for example if `xarray` installs 100 things, but I want to be able to depend on `xarray-core`

1. cytoolz is basically faster toolz.
2. The only dependency ctoolz has is toolz.

conda-forge is about having things work well out of the box. I think cytoolz hits that "just right"

Reply:
This is probably my fault for not reviewing PR ( https://github.com/conda-forge/dask-feedstock/pull/49 ) sooner. I've take a look and it seems basically fine. Maybe that will make things more straightforward here.